### DISCUSSION TITLE
Running Velociraptor on docker via AWS ECS

### DISCUSSION THREAD

**Discussion by dgilmore82** [Velocidex/velociraptor#2805] [Q&A] [answered]
Hello,

Has anyone ran Velociraptor on docker container via this link [velociraptor-docker](https://github.com/weslambert/velociraptor-docker)?  And if so, is it ok to run Velociraptor as a container being managed by [AWS ECS](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/clusters.html) ? 

If not, could you speak on why Velociraptor should not be a container managed by AWS ECS and if we should just deploy Velociraptor on a EC2 instance?

**Comment by scudette** [comment, marked as answer]
Docker is really good for staging software that has a lot of dependencies as it can separate the different modules like database, web server etc, or it can be useful to install libraries and other dependencies.

I personally never use it for velociraptor because there is no point. Velociraptor does not really have any dependencies so there is no real benefit in isolation from the pod OS.  It's a single binary anyway and you don't need to install libraries etc.

I think you can certainly use Docker but it's kind of pointless in my opinion so I never bother.

**Reply by phish-app** [reply]
Can you expand on this? What would you run it on if you needed it on demand and connected to SSO?

I was going to use fargate.

**Comment by scudette** [comment]
I normally just use an Ubuntu vm . Either always running or spin up on demand.
